lib/haml/filters/growstuff_markdown.rb

```diff
@@ -3,6 +3,8 @@
 module Haml::Filters
   module GrowstuffMarkdown
     CROP_REGEX = /\[([^\[\]]+?)\]\(crop\)/
+    MEMBER_REGEX = /\[([^\[\]]+?)\]\(member\)/
+    MEMBER_AT_REGEX = /(\@[^\[ \n]+)/
```

Comment:
Do you mean "an `@` followed by one or more non-whitespace characters"? I'd express that as `/@\S+/`. But actually, we're more restrictive about the characters allowed in usernames - we only allow [letters, numbers and underscores](https://github.com/Growstuff/growstuff/blob/dev/app/models/member.rb#L76). What you *really* want is `/@\w+/`.